docs/the-new-architecture/pillars-fabric-components.md

```diff
@@ -531,6 +531,11 @@ cd android
 
 This script first adds the package to the app, in the same way iOS does. Then, after moving to the `android` folder, it invokes a `gradlew` task to generate the codegen.
 
+:::note
+To run the codegen, you need to enable the **New Architecture** in the Android app. This can be done by opening the `gradle.properties` files and by switching the `newArchEnabled` property from `false` to `true`.
```

Comment:
```suggestion
To run the CodeGen, you need to enable the **New Architecture** in the Android app. This can be done by opening the `gradle.properties` files and by switching the `newArchEnabled` property from `false` to `true`.
```